src/Generator.php

```diff
@@ -229,7 +236,9 @@ protected function setProcessors(OpenApiGenerator $generator): void
             $processors[] = $processor;
             if ($processor instanceof \OpenApi\Processors\BuildPaths) {
                 foreach ($processorClasses as $customProcessor) {
-                    $processors[] = new $customProcessor();
+                    $customProcessorObj = new $customProcessor();
+                    $customProcessorObj->config = $this->config;
```

Comment:
processors in `swagger-php` have no common interface, so this will only work if client will add `config` prop to it's processor. This will create a dynamic prop in the class now, but this is going to be deprecated in php8.2

So we need to thing about some internal interface, which has defined `setConfig()` method and only if custom processor implements it - do the setting.

Reply:
I have had similar thoughts, but since processors can also be callables this didn't get very far.
That's why I opened for checking for matching setXXX() setter methods...